index.js

```diff
@@ -21,7 +21,7 @@ const HOSTNAME_IP_REGEX = /[^0-9a-zA-Z\-.]/g
 const HOSTNAME_PATH = '/etc/logdna-hostname'
 
 // Variables
-var config = require('./lib/config')
+let config = require('./lib/config')
```

Comment:
'config.js' since we're updating this line

Reply:
sure thing